src/core/caniuse.js

```diff
@@ -0,0 +1,289 @@
+  try {
+    const json = await getJson(url);
+    showData(key, json.stats, conf, parent);
+    cache.set(key, { stats: json.stats, cacheTime: new Date() })
+      .catch(err => console.error("[core/caniuse] (Could not cache)", err));
+  } catch (err) {
+    showError(err, key, placeholder);
+  }
+}
+
+/**
+ * creates a placeholder while the API fetches results
+ * @param  {String} key     API feature name
+ * @param  {Node} parent    where to create placeholder
+ * @return {Node}           inserted placeholder
+ */
+function createPlaceholder(key, parent) {
+  const placeholder = hyperHTML`
+    <div class="caniuse" id="${`caniuse-${key}`}">
+      <dt class="caniuse-title">
+        Can I Use this API? (${key})
+      </dt>
+      <dd>fetching data from caniuse.com... </dl>
+    </div>`;
+  return parent.parentNode.insertBefore(placeholder, parent.nextSibling);
+}
+
+// TODO: replace with fetch in core/github ?
+async function getJson(url) {
+  const response = await window.fetch(url);
+  if (!response.ok) {
+    if (response.status === 404) {
+      console.error(`The resource ${url} could not be found (HTTP 404)`);
+      throw new Error("Could not fetch GitHub resource (HTTP 404)");
+    }
+    const errorMsg = `GitHub Response not OK. Probably exceeded request limit. (HTTP ${response.status})`;
+    console.error(`${errorMsg}. Resource = ${url}`);
+    throw new Error(errorMsg);
+  }
+  return await response.json();
+}
+
+function showError(err, key, placeholder) {
+  const permalink = `http://caniuse.com/#feat=${key}`;
+  hyperHTML.bind(placeholder.querySelector("dd"))`
+    Error [core/caniuse]: ${err.message}.
+    <br>Please check directly on <a href="${permalink}">${permalink}</a>.`;
+}
+
+const BROWSERS = { // browser name dictionary
+  chrome: "Chrome",
+  firefox: "Firefox",
+  ie: "IE",
+  edge: "Edge",
+  android: "Android",
+  safari: "Safari",
+  opera: "Opera",
+  bb: "Blackberry",
+  and_uc: "UC (Android)",
+  and_ff: "Firefox (Android)",
+};
+
+/**
+ * render the canIUse support table
+ * @param  {String} key       API name
+ * @param  {Object} stats     CanIUse API results
+ * @param  {Node} parent      where to render table
+ * @param  {Object} conf      respecConfig.caniuse
+ */
+function showData(key, stats, conf, parent) {
+  let browsers = conf.browsers;
+  if (conf.browsers === "ALL") {
+    browsers = Object.keys(stats);
+  }
+
+  // utils
+  const canIUseId = `caniuse-${key}`;
+  const permalink = `http://caniuse.com/#feat=${key}`;
+
+  const validBrowsers = browsers.filter(b => b in stats);
+  if (validBrowsers.length !== browsers.length) {
+    pub("warn", `Unsupported value(s) in \`conf.caniuse.browsers\`.
+    The following were ignored: \`[${browsers.filter(b => !(b in stats))}]\``);
+  }
+
+  // render the support table
+  const caniuse = hyperHTML`
+    <div class="caniuse" id="${canIUseId}">
+      <dt>Can I Use this API? (${key})</dt>
+      <div class="caniuse-stats">
```

Comment:
this should be a `dd`

Reply:
`<dd class="caniuse-stats">` ?

Reply:
I've finally understood how to read the GitHub contexts! LOL. (need to read on files page instead on conversation one)